src/api/lib/tasks/cleanup_source_targets_request_action.rake

```diff
@@ -0,0 +1,28 @@
+desc('Insert source_project_id and source_package_id in bs_request_actions')
+task(backfill_source: :environment) do
+  bs_request_actions = BsRequestAction.where(source_project_id: nil, source_package_id: nil).where('source_project IS NOT NULL OR source_package IS NOT NULL')
+  bs_request_actions.find_each do |action|
```

Comment:
I think you better chunk the iterator, or it will try to load the whole collection of elements in memory...